app/policies/application_policy.rb

```diff
@@ -0,0 +1,23 @@
+class ApplicationPolicy
+  attr_reader :user, :document_class
+  def initialize(user, document_class)
+    @user = user
+    @document_class = document_class
+  end
+
+  def user_organisation_owns_document_type?
+    document_class.organisations.include?(user.organisation_content_id)
+  end
+
+  def departmental_editor
+    user_organisation_owns_document_type? && user.permissions.include?('editor')
+  end
+
+  def writer
+    user_organisation_owns_document_type?
+  end
+
+  def gds_editor
+    user.gds_editor?
+  end
+end
```

Comment:
Thanks for pointing this out, I'm adding it in this [commit](https://github.com/alphagov/specialist-publisher-rebuild/pull/730/commits/49ec86c9811feccee248c4ef6b673453ff613416).